modules/seqexec/server/src/main/scala/seqexec/server/gmos/GmosInstrumentActions.scala

```diff
@@ -105,50 +105,37 @@ class GmosInstrumentActions[F[_]: MonadError[?[_], Throwable]: Concurrent: Logge
   def oneSubExposure(
     fileId:    ImageFileId,
     sub:       NSSubexposure,
-    rows:      Int,
     positions: Vector[NSPosition],
     env:       ObserveEnvironment[F],
     post:      (Stream[F, Result[F]], ObserveEnvironment[F]) => Stream[F, Result[F]]
   ): Stream[F, Result[F]] = {
-    val rowsToShuffle = Gmos.rowsToShuffle(sub.stage, rows)
     val nsPositionO   = positions.find(_.stage === sub.stage)
-    // Configure GMOS rows
-    Stream
-      .eval(
-        inst
-          .configureShuffle(rowsToShuffle)
```

Comment:
Core change 2. We shouldn't set rows to shuffle in between subexposures. Discovered experimentally